cypress/tsconfig.json

```diff
@@ -2,7 +2,8 @@
   "compilerOptions": {
     "target": "es2021",
     "lib": ["es2021", "dom"],
-    "types": ["node", "cypress"]
+    "moduleResolution": "node",
+    "types": ["node", "cypress", "monaco-editor"]
```

Comment:
I think `types` here are for global definitions.
Is that necessary for your case? Or could you import these on the few files where the monaco-editor is referenced?

Reply:
Good question, I can't recall now how it ended up like this. I will double check and either fix or motivate.